By default tests are run on build. Normally, Alpine tests are invoked
separately in a check stage. To allow doing that this patch disable the
automatic test invocations during build.

diff -upr OpenBLAS-0.3.15.orig/Makefile OpenBLAS-0.3.15/Makefile
--- OpenBLAS-0.3.15.orig/Makefile	2021-06-18 22:19:41.361760258 +0200
+++ OpenBLAS-0.3.15/Makefile	2021-06-18 22:19:48.631738269 +0200
@@ -39,7 +39,7 @@ SUBDIRS_ALL = $(SUBDIRS) test ctest utes
 .PHONY : all libs netlib $(RELA) test ctest shared install
 .NOTPARALLEL : all libs $(RELA) prof lapack-test install blas-test
 
-all :: libs netlib $(RELA) tests shared
+all :: libs netlib $(RELA) shared
 	@echo
 	@echo " OpenBLAS build complete. ($(LIB_COMPONENTS))"
 	@echo
